psi4/src/psi4/lib3index/df_helper.h

```diff
@@ -335,19 +348,34 @@ class DF_Helper {
     std::map<std::string, std::vector<double>> transf_core_;
 
     // => transformation machinery <=
-    void transform_core();
-    void transform_disk();
+    void put_transformations_Qpq(int naux, int begin, int end,  
+        int wsize, int bsize, double* Fp, int ind, bool bleft);
+    void put_transformations_pQq(int naux, int begin, int end, int block_size, int bcount, 
+        int wsize, int bsize, double* Np, double* Fp, int ind, bool bleft);
     std::vector<std::pair<std::string, size_t>> sorted_spaces_;
     std::vector<std::string> order_;
     std::vector<std::string> bspace_;
     std::vector<size_t> strides_;
 
     // => FILE IO maintenence <=
-    struct stream {
-        FILE* fp;
-        std::string op;
-    };
-    std::map<std::string, stream> file_status_;
+    typedef struct StreamStruct {
```

Comment:
One thing id like to keep in mind while we are doing new IO tech is to always ask the question "Can I port this to HDF5 easily"? Curious on your thoughts here.

Reply:
The DF_Helper IO tech is mostly the same, I just organized it differently.  I am not sure how this impacts portability to HDF5.  I would be happy to help move DF_Helper IO to a new regime once we get there.